### DISCUSSION TITLE
Pass data to txt2img UI page with external program

### DISCUSSION THREAD

**Discussion by ChiaYen-Kan** [vladmandic/sdnext#2922] [Q&A]
i has prompt/negative prompt/generate parameters store in my program

i read automatic1111 API doc, the tex2img API is generate image immediately, but i just want to fill prompt/negative prompt/generate parameters to txt2img UI page, don't generate image immediately

how i can pass the data to txt2img UI page from external program?

**Comment by vladmandic** [comment]
you can either use api or ui, but you cannot (and should not) use api to control ui.
